bundle/phases/initialize.go

```diff
@@ -119,6 +119,11 @@ func Initialize(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 		// ApplyPresets through ResourceProcessor.
 		resourcemutator.ApplyTargetMode(),
 
+		// Reads (typed): b.SyncRoot (checks if bundle root is in /Workspace/)
+		// Updates (typed): b.SyncRoot (replaces with extension-aware path when running on Databricks Runtime)
+		// Configure use of WSFS for reads if the CLI is running on Databricks.
+		mutator.ConfigureWSFS(),
+
```

Comment:
The PR summary explains:
> ConfigureWSFS mutator call is moved to be executed before ProcessStaticResources resource mutator, so that the latter has access to the Workspace File System client

But I don't that the new mutator is hooked up to `ProcessStaticResources`.

Reply:
correct. this bit of description is a leftover from the original implementation. will change once we have the final solution

Reply:
PR description is now updated